docs/admin/machine.rst

```diff
@@ -51,8 +50,10 @@ AWS
 Amazon Translate is a neural machine translation service for translating text
 to and from English across a breadth of supported languages.
 
-To enable this service, add ``weblate.machinery.aws.AWSTranslation`` to
-:setting:`MT_SERVICES`, install the `boto3` module and set the settings.
+1. Turn on this service by adding ``weblate.machinery.aws.AWSTranslation`` to
+:setting:`MT_SERVICES`.
+
+2. Install the `boto3` module and configure it.
```

Comment:
Is it the module that needs to be configured?

Reply:
No, just Weblate settings.